Lib/test/test_future_stmt/test_future.py

```diff
@@ -25,57 +25,71 @@ def check_syntax_error(self, err, basename, lineno, offset=1):
         self.assertEqual(err.offset, offset)
 
     def test_future1(self):
-        with import_helper.CleanImport('future_test1'):
-            from test import future_test1
+        with import_helper.CleanImport('test.test_future_stmt.future_test1'):
+            from test.test_future_stmt import future_test1
             self.assertEqual(future_test1.result, 6)
 
     def test_future2(self):
-        with import_helper.CleanImport('future_test2'):
-            from test import future_test2
+        with import_helper.CleanImport('test.test_future_stmt.future_test2'):
+            from test.test_future_stmt import future_test2
             self.assertEqual(future_test2.result, 6)
 
-    def test_future3(self):
-        with import_helper.CleanImport('test_future3'):
-            from test import test_future3
+    def test_future_single_import(self):
+        with import_helper.CleanImport(
+            'test.test_future_stmt.test_future_single_import',
+        ):
+            from test.test_future_stmt import test_future_single_import
+
+    def test_future_multiple_imports(self):
+        with import_helper.CleanImport(
+            'test.test_future_stmt.test_future_multiple_imports',
+        ):
+            from test.test_future_stmt import test_future_multiple_imports
+
+    def test_future_multiple_features(self):
+        with import_helper.CleanImport(
+            "test.test_future_stmt.test_future_multiple_features",
+        ):
+            from test.test_future_stmt import test_future_multiple_features
 
     def test_badfuture3(self):
```

Comment:
This one tests that a typo is catched properly, that an invalid name raises SyntaxError: ``from __future__ import rested_snopes``.

Maybe: ``test_unknown_feature``. The doc calls ``__future__`` names as "features": https://docs.python.org/dev/library/__future__.html